src/components/Input/inputwithlabel.style.css

```diff
@@ -1,39 +1,29 @@
 .input {
     display: flex;
     flex-direction: column;
-    position: relative;
-    padding-bottom: 2.6rem;
+    gap:.5rem;
 
   }
   .label {
     color:white;
     font-size: 1.1rem;
-    position: absolute;
-    top: -2rem;
     font-family: sans-serif;;
   }
   
-  ::placeholder {
+  .input::placeholder {
     font-size: 0.9rem;
     font-family: sans-serif;
   }
   
-  input,
-  textarea {
+ .inputBar {
     padding: 0.7rem;
     padding-left: 1rem;
-    width: 18rem;
-    height: 1.5rem; 
     border-radius: 0.5rem;
     margin-top: 0.2rem;
     color: white; 
     background-color: #222; 
     border: 1px solid black; 
   }
   
-  textarea {
-    height: 3rem; 
-    resize: vertical;
-  }
```

Comment:
`resize: vertical` shouldn't be removed. It is important so the text area doesn't move freely out of its container.
And it is a good practice to also set min-height and max-height to restrict its resizing even more.